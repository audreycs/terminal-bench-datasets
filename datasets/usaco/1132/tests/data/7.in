20 20
qrjeigp yypsyly kwjoukt qkvaygg jwhbnsh yzywxor xzrhxax tnsmxom wwshsak isrgtse qgvnhmw epuxcmj bcxxyqg rssbmkh gcnyklm rzkxoew stiuxft wxcpruq eezmiok uzynzjk
bcxxyqg eezmiok epuxcmj gcnyklm isrgtse jwhbnsh kwjoukt qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wwshsak wxcpruq xzrhxax yypsyly yzywxor
epuxcmj qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wxcpruq xzrhxax bcxxyqg eezmiok gcnyklm isrgtse jwhbnsh kwjoukt qgvnhmw wwshsak yypsyly yzywxor
eezmiok epuxcmj gcnyklm jwhbnsh qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wwshsak wxcpruq xzrhxax yypsyly bcxxyqg isrgtse kwjoukt yzywxor
epuxcmj rssbmkh uzynzjk bcxxyqg eezmiok gcnyklm isrgtse jwhbnsh kwjoukt qgvnhmw qkvaygg qrjeigp rzkxoew stiuxft tnsmxom wwshsak wxcpruq xzrhxax yypsyly yzywxor
epuxcmj rssbmkh rzkxoew uzynzjk bcxxyqg eezmiok gcnyklm isrgtse jwhbnsh kwjoukt qgvnhmw qkvaygg qrjeigp stiuxft tnsmxom wwshsak wxcpruq xzrhxax yypsyly yzywxor
epuxcmj qkvaygg rssbmkh rzkxoew stiuxft uzynzjk wxcpruq bcxxyqg eezmiok gcnyklm isrgtse jwhbnsh kwjoukt qgvnhmw qrjeigp tnsmxom wwshsak xzrhxax yypsyly yzywxor
uzynzjk bcxxyqg eezmiok epuxcmj gcnyklm isrgtse jwhbnsh kwjoukt qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom wwshsak wxcpruq xzrhxax yypsyly yzywxor
eezmiok epuxcmj jwhbnsh qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wxcpruq xzrhxax bcxxyqg gcnyklm isrgtse kwjoukt wwshsak yypsyly yzywxor
eezmiok epuxcmj qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wxcpruq xzrhxax bcxxyqg gcnyklm isrgtse jwhbnsh kwjoukt wwshsak yypsyly yzywxor
epuxcmj qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wxcpruq xzrhxax eezmiok jwhbnsh bcxxyqg gcnyklm isrgtse kwjoukt wwshsak yypsyly yzywxor
epuxcmj qkvaygg qrjeigp rssbmkh rzkxoew stiuxft uzynzjk wxcpruq xzrhxax bcxxyqg eezmiok gcnyklm isrgtse jwhbnsh kwjoukt qgvnhmw tnsmxom wwshsak yypsyly yzywxor
epuxcmj qkvaygg qrjeigp rssbmkh rzkxoew stiuxft uzynzjk wxcpruq xzrhxax eezmiok qgvnhmw tnsmxom bcxxyqg gcnyklm isrgtse jwhbnsh kwjoukt wwshsak yypsyly yzywxor
eezmiok epuxcmj jwhbnsh qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wwshsak wxcpruq xzrhxax bcxxyqg gcnyklm isrgtse kwjoukt yypsyly yzywxor
eezmiok epuxcmj gcnyklm jwhbnsh qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wwshsak wxcpruq xzrhxax bcxxyqg isrgtse kwjoukt yypsyly yzywxor
bcxxyqg eezmiok epuxcmj gcnyklm isrgtse jwhbnsh qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wwshsak wxcpruq xzrhxax yypsyly yzywxor kwjoukt
eezmiok epuxcmj gcnyklm isrgtse jwhbnsh qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wwshsak wxcpruq xzrhxax yypsyly yzywxor bcxxyqg kwjoukt
epuxcmj qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom uzynzjk wxcpruq xzrhxax bcxxyqg eezmiok gcnyklm isrgtse jwhbnsh kwjoukt wwshsak yypsyly yzywxor
epuxcmj uzynzjk bcxxyqg eezmiok gcnyklm isrgtse jwhbnsh kwjoukt qgvnhmw qkvaygg qrjeigp rssbmkh rzkxoew stiuxft tnsmxom wwshsak wxcpruq xzrhxax yypsyly yzywxor
epuxcmj qkvaygg rssbmkh rzkxoew stiuxft uzynzjk eezmiok jwhbnsh qgvnhmw qrjeigp tnsmxom wwshsak wxcpruq xzrhxax bcxxyqg gcnyklm isrgtse kwjoukt yypsyly yzywxor
epuxcmj qkvaygg rssbmkh rzkxoew uzynzjk bcxxyqg eezmiok gcnyklm isrgtse jwhbnsh kwjoukt qgvnhmw qrjeigp stiuxft tnsmxom wwshsak wxcpruq xzrhxax yypsyly yzywxor
