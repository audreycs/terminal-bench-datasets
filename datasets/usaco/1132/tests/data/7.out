B00100?0000101011101
1B011011101101111111
11B11111111111111111
000B0000000101010001
1001B011001101011111
11011B111?1101111111
?00100B0000101011101
1001001B000101011101
10011011B01101011111
11011?111B1101111111
1001001100B101011101
00000000000B00000001
110111111111B1111111
0000000000010B000001
10011011101101B11111
000000000001010B0001
0001000000010101B001
00010000000101011B01
100100110011010111B1
0000000000000000000B
